### PR TITLE
Add warning to not commit env files

### PR DESCRIPTION
**Pull request by Dentrax**
Signed-off-by: Furkan <furkan.turkal@trendyol.com>

## Description

This PR adds an important security warning message for the `.env` file creation steps. Accidentally committing `.env` file  to a git repo is a mistake. Adding some warning messages, so devs don't forget.

### CODE REVIEW

File: src/content/developers/tutorials/hello-world-smart-contract/index.md [hunk 1]
```diff
@@ -233,6 +233,10 @@ PRIVATE_KEY = "your-metamask-private-key"
 
 To actually connect these to our code, we’ll reference these variables in our `hardhat.config.js` file on step 13.
 
+<InfoBanner isWarning={true}>
+  Don't commit `.env`: Please make sure to never share or expose your `.env` file with anyone as you can compromise your secrets in doing so. If you are using version control, ensure that you will never commit the file to upstream.
```
Comment (minimalsm) [inline src/content/developers/tutorials/hello-world-smart-contract/index.md:237]: I'd use stronger language here. 

Also backticks won't work inside HTML. Added <pre> tags instead.

```suggestion
Don't commit <pre>.env</pre>! Please make sure never to share or expose your <pre>.env</pre> file with anyone, as you are compromising your secrets in doing so. If you are using version control, add your <pre>.env</pre> to a <a href="https://git-scm.com/docs/gitignore">gitignore</a> file.
```

File: src/content/developers/tutorials/how-to-write-and-deploy-an-nft/index.md [hunk 2]
```diff
@@ -221,21 +221,25 @@ First, install the dotenv package in your project directory:
...
-Your .env should now look like this:
+Your `.env` should now look like this:
 
     API_URL="https://eth-ropsten.alchemyapi.io/v2/your-api-key"
     PRIVATE_KEY="your-metamask-private-key"
 
 To actually connect these to our code, we’ll reference these variables in our hardhat.config.js file on step 13.
 
+<InfoBanner isWarning={true}>
+  Don't commit `.env`: Please make sure to never share or expose your `.env` file with anyone as you can compromise your secrets in doing so. If you are using version control, ensure that you will never commit the file to upstream.
```
Comment (minimalsm) [inline src/content/developers/tutorials/how-to-write-and-deploy-an-nft/index.md:240]:
```suggestion
Don't commit <pre>.env</pre>! Please make sure never to share or expose your <pre>.env</pre> file with anyone, as you are compromising your secrets in doing so. If you are using version control, add your <pre>.env</pre> to a <a href="https://git-scm.com/docs/gitignore">gitignore</a> file.
```

File: src/content/developers/tutorials/sending-transactions-using-web3-and-alchemy/index.md [hunk 3]
```diff
@@ -109,6 +109,10 @@ API_URL = "your-api-url"
 PRIVATE_KEY = "your-private-key"
 ```
 
+<InfoBanner isWarning={true}>
+  Don't commit `.env`: Please make sure to never share or expose your `.env` file with anyone as you can compromise your secrets in doing so. If you are using version control, ensure that you will never commit the file to upstream.
```
Comment (minimalsm) [inline src/content/developers/tutorials/sending-transactions-using-web3-and-alchemy/index.md:113]:
```suggestion
Don't commit <pre>.env</pre>! Please make sure never to share or expose your <pre>.env</pre> file with anyone, as you are compromising your secrets in doing so. If you are using version control, add your <pre>.env</pre> to a <a href="https://git-scm.com/docs/gitignore">gitignore</a> file.
```

**Review by minimalsm — CHANGES_REQUESTED** [review]
Hey @Dentrax, I think this is an awesome idea. I had similar concerns when reviewing a tutorial a few days ago.

Made a suggestion to use slightly stronger language (are exposing secrets vs might be exposing secrets) and to let users know to use a gitignore file.

**Comment by Dentrax** [comment]
Thank you for reviews, @minimalsm! 🙏 

Is there somewhere that I forgot to add this warning? Should I squash my commits?

File: src/content/developers/tutorials/hello-world-smart-contract/index.md [hunk 4]
```diff
@@ -233,6 +233,10 @@ PRIVATE_KEY = "your-metamask-private-key"
 
 To actually connect these to our code, we’ll reference these variables in our `hardhat.config.js` file on step 13.
 
+<InfoBanner isWarning={true}>
+Don't commit <pre>.env</pre>! Please make sure never to share or expose your <pre>.env</pre> file with anyone, as you are compromising your secrets in doing so. If you are using version control, add your <pre>.env</pre> to a <a href="https://git-scm.com/docs/gitignore">gitignore</a> file.
```
Comment (minimalsm) [inline src/content/developers/tutorials/hello-world-smart-contract/index.md:237]:
```suggestion
Don't commit <code>.env</code>! Please make sure never to share or expose your <code>.env</code> file with anyone, as you are compromising your secrets in doing so. If you are using version control, add your <code>.env</code> to a <a href="https://git-scm.com/docs/gitignore">gitignore</a> file.
```

File: src/content/developers/tutorials/how-to-write-and-deploy-an-nft/index.md [hunk 5]
```diff
@@ -221,21 +221,25 @@ First, install the dotenv package in your project directory:
...
-Your .env should now look like this:
+Your `.env` should now look like this:
 
     API_URL="https://eth-ropsten.alchemyapi.io/v2/your-api-key"
     PRIVATE_KEY="your-metamask-private-key"
 
 To actually connect these to our code, we’ll reference these variables in our hardhat.config.js file on step 13.
 
+<InfoBanner isWarning={true}>
+Don't commit <pre>.env</pre>! Please make sure never to share or expose your <pre>.env</pre> file with anyone, as you are compromising your secrets in doing so. If you are using version control, add your <pre>.env</pre> to a <a href="https://git-scm.com/docs/gitignore">gitignore</a> file.
```
Comment (minimalsm) [inline src/content/developers/tutorials/how-to-write-and-deploy-an-nft/index.md:240]:
```suggestion
Don't commit <code>.env</code>! Please make sure never to share or expose your <code>.env</code> file with anyone, as you are compromising your secrets in doing so. If you are using version control, add your <code>.env</code> to a <a href="https://git-scm.com/docs/gitignore">gitignore</a> file.
```

File: src/content/developers/tutorials/sending-transactions-using-web3-and-alchemy/index.md [hunk 6]
```diff
@@ -109,6 +109,10 @@ API_URL = "your-api-url"
 PRIVATE_KEY = "your-private-key"
 ```
 
+<InfoBanner isWarning={true}>
+Don't commit <pre>.env</pre>! Please make sure never to share or expose your <pre>.env</pre> file with anyone, as you are compromising your secrets in doing so. If you are using version control, add your <pre>.env</pre> to a <a href="https://git-scm.com/docs/gitignore">gitignore</a> file.
```
Comment (minimalsm) [inline src/content/developers/tutorials/sending-transactions-using-web3-and-alchemy/index.md:113]:
```suggestion
Don't commit <code>.env</code>! Please make sure never to share or expose your <code>.env</code> file with anyone, as you are compromising your secrets in doing so. If you are using version control, add your <code>.env</code> to a <a href="https://git-scm.com/docs/gitignore">gitignore</a> file.
```

**Review by minimalsm — CHANGES_REQUESTED** [review]
TIL `<pre>` tags split across multiple lines 😅

Screenshot 2022-02-22 at 11 45 26

**Comment by minimalsm** [comment]
> Is there somewhere that I forgot to add this warning? Should I squash my commits?

@Dentrax nope, this is fine as is, thanks. Just waiting on this building to make sure I fixed the bug I introduced and I'll merge this 🚀🚀🚀🚀

**Comment by minimalsm** [comment]
Fixed, thanks again @Dentrax 

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And remember to [claim your POAP](https://ethereum.org/en/contributing/#poap) also 🥳! We're currently working on our 2022 POAPs and they'll be released soon.

**Review by minimalsm — APPROVED** [review]

**Pull request merged**